users/333fred/rules.mk

```diff
@@ -7,3 +7,4 @@ endif
 TAP_DANCE_ENABLE=yes
 NKRO_ENABLE = yes
 PERMISSIVE_HOLD = yes
+LTO_ENABLE = yes
```

Comment:
You may not want this applying to everything, as it can cause issues on ChibiOS (ARM).  Though, that may not be an issue for you, at least, as of yet.